csss-site/src/events/workshops/templates/workshops/main.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+{% extends "csss/header.html" %}
+{% block content %}
+{{ block.super }}
+        <h2 class="title is-size-2" style="padding-left: 1em;">Computing Science Student Society Workshops</h2>
+        <div style="padding-left: 2em;">Note: Most of these workshop slides were part of various levels of demos. Parts of slides may be left empty or leave questions unanswered for that reason.</div><br><br>   
+
+        <h3 class="title" style="padding-left: 1em;">Git Workshops</h3>
+	<div style="padding-left: 2em;">Learn about Git and GitHub, what they are, how to use them in both the terminal and the Visual Studio Code.</div><br>
+        <div style="text-align:center;">
+		<iframe src="https://drive.google.com/file/d/1pf7oIyWrIUJSZJctnitKTAIBw26ZmitL/preview" width="800" height="480" allow="autoplay"></iframe>
+	</div><br>
+
+	<h3 class="title" style="padding-left: 1em;">Linux Workshops</h3>
+	<div style="padding-left: 2em;">Learn all about Linux.</div><br>                                                       
+        <div style="text-align:center;"> 
+		<iframe src="https://drive.google.com/file/d/1jXvkVarqTZym9Z8Hw6CgJAKLnNRucFYE/preview" width="800" height="600" allow="autoplay"></iframe>
```

Comment:
I don't think this belongs in the CSSS Team Drive since the website is not keeping track of it to ensure its always publicly available.